src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OpenTelemetryProtocolExporterEventSource.cs

```diff
@@ -55,10 +55,10 @@ public void CouldNotTranslateActivity(string className, string methodName)
         this.WriteEvent(3, className, methodName);
     }
 
-    [Event(4, Message = "Unknown error in export method: {0}", Level = EventLevel.Error)]
-    public void ExportMethodException(string ex)
+    [Event(4, Message = "Unknown error in export method. Message: '{0}'. IsRetry: {1}", Level = EventLevel.Error)]
```

Comment:
AOT IL2026 could be suppressed here as the params to this method are all [primitive types](https://github.com/dotnet/runtime/blob/ab2b80d06dd4d997df5ffa72a3c4a99cb36ccbff/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/Statics.cs#L484-L574).